### ISSUE TITLE
Revisão por pesquisadores especialistas no Ensino de Frações

### ISSUE THREAD

**Issue by simas0** [livro-aberto/fracoes_livro_piloto#143]
Como determinado na reunião de 20 de dezembro de 2016 vamos solicitar a revisão do material a pesquisadores renomados no Ensino de Frações. Escrevi a seguinte mensagem que deve ser enviada aos pesquisadores após a inclusão das figuras corrigidas e a geração do novo pdf. Gostaria que vocês complementassem a lista de pesquisadores e avaliassem se a mensagem é apropriada. Pretendo enviar o e-mail na quinta-feira desta semana.

> Caros colegas, 
> 
> escrevemos um livro didático de frações para professores e estudantes do Ensino Fundamental  baseado em pesquisas científicas sobre o ensino deste assunto. Os senhores foram identificados como pesquisadores especialistas no Ensino de Frações e, por isso, vimos solicitar-lhes a gentileza de nos fornecer suas impressões sobre o nosso material.
> 
> Este material deve ser utilizado em diversas escolas públicas de Ensino Fundamental neste ano de 2017 e por isso solicitamos que a sua avaliação nos seja enviada por e-mail até o dia 10 de fevereiro.
> 
> Informamos também que a produção deste livro foi  financiada pela OBMEP/IMPA, é uma produção Livro Aberto e a ele foi atribuída a licença Creative Commons by-sa-nc, que permite impressão, divulgação e derivações do material desde que seu uso não seja comercial, mantenha-se a mesma licença e que seja atribuída a fonte (autores, OBMEP/IMPA e Livro Aberto).
> 
> O livro pode ser baixado pelo link a seguir:
> (https://www.umlivroaberto.com/wp/?page_id=89)
> 
> Cordialmente,
> Fabio Simas
> Coordenador do Projeto Livro Aberto de Matemática

**Pesquisadores do tema Frações.** 
Nilza Bertoni 
Lourdes Onuchic 
Hélia Margarida Ventura 
Hélia Pinto 
Terezinha Nunes 
Elizabeth Belfort

**Comment by hjbortol** [comment]
Eu inverteria a ordem na carta: falaria primeiro do projeto, para depois falar do livro de frações para depois fazer o convite. Eu também incluiria uma informação dizendo que os avaliadores receberão um certificado de parecerista ad hoc.

O que acham?

> Caros colegas,
> 
> O Projeto "Um Livro Aberto" (http://umlivroaberto.com) é um esforço de professores e alunos em produzir, de maneira colaborativa, livros didáticos de matemática com excelência acadêmica e licença aberta.  O objetivo é garantir livre reprodução, distribuição, modificações e derivações dos materiais produzidos. O projeto conta com o apoio financeiro da OBMEP e do IMPA.
> 
> O primeiro livro escrito pelo projeto trata-se de um livro didático de frações para professores e estudantes do Ensino Fundamental cuja elaboração foi apoiada fortemente em pesquisas científicas sobre o ensino deste assunto. Seus nomes foram identificados como pesquisadoras especialistas no Ensino de Frações e, por isso, vimos solicitar-lhes a gentileza de nos fornecer suas impressões sobre o nosso material. Pelo trabalho, o IMPA e a OBMEP fornecerão um certificado de parecerista ad hoc.
> 
> Este material deve ser utilizado em diversas escolas públicas de Ensino Fundamental neste ano de 2017 e por isso solicitamos que a sua avaliação nos seja enviada por e-mail até o dia 10 de fevereiro.
>    
> O livro pode ser baixado pelo link a seguir: (https://www.umlivroaberto.com/wp/?page_id=89)
> 
> Cordialmente,
> Fabio Simas
> Coordenador do Projeto Livro Aberto de Matemática
>

**Comment by gutosurrex** [comment]
Concordo com as sugestões do Humberto. Além disso incluiria um link para umlivroaberto. Pois alguns podem se interessar mais sobre o projeto, ou até mesmo querer saber mais antes de se comprometer.

A licença ficou essa mesma?

**Comment by simas0** [comment]
Obrigado pelas colaboração, @hjbortol! 
@gutosurrex, esta foi a licença acordada com o IMPA para o piloto. Os direitos dos usuários serão ampliados em 2026.

**Comment by cydara** [comment]
Gentes, algum parecerista já enviou seus comentários?

**Comment by simas0** [comment]
Eu dei o prazo até o dia 14 de março. Hoje escrevi pedindo uma confirmação de recebimento e disponibilidade em colaborar. Estou recebendo respostas.
